funnel/assets/js/utils/form_widgets.js

```diff
@@ -60,7 +60,7 @@ export const Widgets = {
     };
     this.activateToggleSwitch(checkboxId, onSuccess);
   },
-  activate_select2() {
+  activateSelect2Focus() {
     /* Upgrade to jquery 3.6 select2 autofocus isn't working. This is to fix that problem.
       select2/select2#5993  */
```

Comment:
This ticket is marked as closed as jQuery 3.7 fixes it. Do we still need this patch?

Reply:
Will open a new PR to upgrade to jQuery 3.7 and remove this fix